src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.birth();
```

Comment:
it's redundant

Reply:
sorry for late response. so i need to put this constructor to each animal or what? cant actually understand what i should fix here(